sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestRegisteredMappersWork.java

```diff
@@ -106,6 +116,25 @@ public void testBeanMapperFactory() throws Exception
     }
 
     @Test
+    public void testBeanMapperFactoryDefaultMethod() throws Exception
+    {
+        BeanMappingDao bdb = db.getSharedHandle().attach(BeanMappingDao.class);
+        bdb.createBeanTable();
+
+        Bean lima = new Bean();
```

Comment:
Heh, can't take credit for that--it's just a copypasta and modification of a preexisting test. 😆